src/components/TransactionsList.js

```diff
@@ -27,8 +31,8 @@ const TransactionsList = () => {
             </h3>
           </th>
         </tr>
-
-        {"... your code here..."}
+        
+        {filterList.map((transaction) => <Transaction posted={transaction.posted_at} description={transaction.description} category={transaction.category} amount={transaction.amount} />) }
```

Comment:
i'd probably just pass the `transaction` object as a prop, then de-structure it on the `Transaction` component.